qualtrics_link/templates/qualtrics_link/_unauthorized_message.html

```diff
@@ -0,0 +1,27 @@
+<h3>
+  We're sorry, your ID doesn't appear to have access to this Qualtrics license.
+</h3>
+<p>
+  <ul>
+    <li>
+      HMS/HSDM affiliates--<a href="https://collaborate.hms.harvard.edu/x/z4Gm">instructions
+      on how to login to your HMS Qualtrics account:
```

Comment:
(looks like the URL times out, as you mentioned -- good catch -- we'll push this through under the assumption that HMS will set up a page there eventually)